### DISCUSSION TITLE
Somehow I always end up in DEV on installing//updating

### DISCUSSION THREAD

**Discussion by LarsVegas76** [Part-DB/Part-DB-server#317] [General]
Please bear with me, if this is a stupid question - I'm fairly new to GITHUB...

When I first installed Part-DB a few weeks ago, the DEV-Version was installed. I followed the "Direct Installation on Debian 11" instructions. After playing around a little, I deleted everything and did a fresh install following the exact same instructions. This time 1.50 Prod was installed. Maybe just "bad timing" on my first install...

Today I updated the installation (using the "Update part" in the instructions). Everything worked fine, but the version is 1.60 DEV now.

Is this a problem? E.g. could this lead into a dead end, where I'm stuck in the DEV path an can not install any updates? Or is it possible to switch to PROD again? If so, how can I do it?

Last but not least: Thanks @jbtronics for the great work! Part-DB is awesome and helps us a lot! THANKS!

Cheers
Lars

**Comment by jbtronics** [comment]
The installation guide comes from a time where there was no stable releases, so the guide was actually missing a crucial step.
`git pull` normally puts you on the latest commit of the `master` branch, which represents the latest development (and which is running on the Demo instance) and might contain some bugs.

To get a stable version, you have to switch to the desired  tagged version, which you can do by running `git checkout v1.5.2` (or similar for other versions). Or you run `git checkout $(git describe --tags $(git rev-list --tags --max-count=1))` to switch to the most recent release.

So running `git checkout $(git describe --tags $(git rev-list --tags --max-count=1))` should fix your problem.  Afterwards you will have to run `composer install -o --no-dev` , `yarn install && yarn build` and `php bin/console cache:clear` to get the right dependencies for the stable version.

I have updated the installation guide with this step.

**Comment by LarsVegas76** [comment]
Thanks for the quick answer! I will give it a shot!

**Comment by LarsVegas76** [comment]
Just to confirm, works like a charm now! Switched to 1.52 without any issues.

Thank you @jbtronics!
